cpp/src/join/mixed_join_semi.cu

```diff
@@ -23,6 +23,7 @@
 #include <cudf/detail/iterator.cuh>
 #include <cudf/detail/null_mask.hpp>
 #include <cudf/detail/utilities/cuda.cuh>
+#include <cudf/hashing/detail/helper_functions.cuh>
```

Comment:
Was this needed but missing before?

Reply:
Yes, we depended on transitive inclusion.